### ISSUE TITLE
Problem when I switch front_facing camera and back_facing camera

### ISSUE THREAD

**Issue by newtonker** [cats-oss/android-gpuimage#135]
Hi, I used this demo recently in my projects. However, I got a problem when I switch camera between front_facing camera and back_facing camera. There will be a very short inverted frame when I switch the camera. Did you notice that? I am wondering the reason, hope to hearing from you.

**Comment by gkwl** [comment]
i encounter this problem too

**Comment by cooliceman** [comment]
@gkwl Your answer works perfect. It solved my problem.

**Comment by filipst** [comment]
Works for me too.

**Comment by jibinjoseph** [comment]
Me too..

**Comment by jibinjoseph** [comment]
Anybody noticed a lag in camera preview?? In my case when i move the phone the preview was showing the frame very slowly.Any fix for this issue please?

**Comment by geeklok** [comment]
@jibinjoseph Indeed, that's because it didn't make use of the OpenGL extension GL_TEXTURE_EXTERNAL_OES provided by Android API >= 15.
